packages/manager/apps/dedicated/client/app/css/less/utils/variables.less

```diff
@@ -152,6 +152,8 @@
 //hover or select line for list, table, dropfownlist
 @defaultlineselectedbackground: @blue;
 
+@defaultlineevenbackground: #f3feff;
```

Comment:
**nitpick:** Avoid using custom colors. You can choose the appropriate one from https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-colors--primary